.github/workflows/update-sdk-docs.yaml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   build:
+    if: github.repository == 'open-feature/openfeature.dev'
```

Comment:
This is unrelated but I was annoyed that this workflow was running (and failing) on my fork. I can move it to a separate PR if necessary though.